### PR TITLE
add lispers with a donation page

### PR DESCRIPTION
**Pull request by vindarel**
Hello lispers,

I added you on this list with the support information I found. Feel free to ask for edits before I merge (or send PRs afterwards), and to suggest anyone else.

Best,
Vincent

cc @ambrevar @cxxxr @froggey @hexstreamsoft @jackdaniel @jmercouris @mfiano @phoe @shinmera @svetlyak40wt @xach

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,4 +1,141 @@
...
+  - book: the Common Lisp Condition System: https://www.apress.com/us/book/9781484261330
+  - Portable Condition System: https://github.com/phoe/portable-condition-system
+  - Common Lisp Ultra Spec: https://github.com/phoe/clus-data
+
+## shinmera - Nicolas Hafner
+
+- github: https://github.com/Shinmera
+- website: https://shinmera.com/
+- donate:
+  - github sponsors: https://github.com/sponsors/Shinmera
```
Comment (Shinmera) [inline README.markdown:93]: I also have a ko-fi page:
https://ko-fi.com/shinmera

File: README.markdown [hunk 2]
```diff
@@ -1,4 +1,141 @@
...
+- projects:
+  - Radiance: https://github.com/sponsors/Shinmera
```
Comment (Shinmera) [inline README.markdown]: This should be https://github.com/shirakumo/radiance, yeah?
Or perhaps even: https://shirakumo.github.io/radiance

File: README.markdown [hunk 3]
```diff
@@ -1,4 +1,141 @@
...
+  - Qtools: https://github.com/Shinmera/qtools
+  - Trial: https://github.com/Shirakumo/trial
+  - Kandria: https://github.com/Shinmera/kandria
```
Comment (Shinmera) [inline README.markdown]: I'd make this be:
- Qtools: https://shinmera.github.io/qtools
- Trial: https://github.com/Shirakumo/trial
- Kandria: https://kandria.com
- Portacle: https://portacle.github.io

**Comment by Shinmera** [comment]
Thanks! I commented on a few changes about my info.

**Comment by vindarel** [comment]
@Shinmera Hello, I added in your edits.

**Comment by jmercouris** [comment]
Froggey's twitter address appears incorrect.

**Comment by vindarel** [comment]
Thanks, corrected.

File: README.markdown [hunk 4]
```diff
@@ -1,4 +1,142 @@
...
+  - Pyx: https://github.com/mfiano/pyx
+  - pngload: https://github.com/bufferswap/pngload
+
+## phoe - Michał "phoe" Herda
+
+- github: https://github.com/phoe
+- website: https://phoe.github.io/
+- mastodon: https://functional.cafe/@phoe/
+- donate:
+  - github sponsors: https://functional.cafe/@phoe/
```
Comment (phoe) [inline README.markdown]: Should be https://github.com/sponsors/phoe

Comment (vindarel) [reply README.markdown]: corrected. I had problems with my copy&paste this day.

**Comment by Ambrevar** [comment]
Thanks @vindarel, great initiative! ;)

**Comment by Hexstream** [comment]
Why did you intentionally mis-@mention me when [you knew what my account name was from the very start](https://github.com/vindarel/lisp-maintainers/tree/ff91277dfedc28cb1ce2af39dfdf27ce863be2c1#hexstreamsoft---jean-philippe-paradis), and did not attempt to rectify even after seeing that my ostensible @mention was clearly not highlighted like the others? Getting the damn notification 2 days ago would surely have been useful.

Thanks for including me in the list, though. I probably don't need to mention that this largely overlaps with [the Common Lispers list](https://common-lispers.hexstreamsoft.com/). Enter `has:funding` in the search field.

Also, [don't take /r/lisp/ feedback too seriously. It is a shithole.](https://twitter.com/HexstreamSoft/status/1308866571557888001) Also see the [previous evidence (thread)](https://twitter.com/HexstreamSoft/status/1112244159115681792).

**Comment by vindarel** [comment]
Hi @Hexstream, sorry for the bad mention, I did not intentionally do it. I possibly copy-pasted badly and did not check the rendered markdown.

I did not know about `has:funding` nor about the search box in your list! Also (it's dumb I know) but I searched "donate" in the page and didn't find a mention.

So yes they overlap, and more than I thought. So, let's say it's better for the lisp developers to have many references rather than one.
Also, your page is better referenced! (see your Alexa ranking!)

Now to the other issue you opened.

**Comment by Hexstream** [comment]
> Hi @Hexstream, sorry for the bad mention, I did not intentionally do it. I possibly copy-pasted badly and did not check the rendered markdown.

Doubtful, but what can I do...

> I searched "donate" in the page and didn't find a mention.

It seems that you were not sufficiently familiar with the [Common Lispers list](https://common-lispers.hexstreamsoft.com/), which might indicate that it could be better referenced.

> Also, your page is better referenced! (see your Alexa ranking!)

Yes, I'm finally [starting to break through](https://twitter.com/HexstreamSoft/status/1308095171087736834) [after years of being censored for no reason](https://twitter.com/HexstreamSoft/status/1309142791579152386) (hexstreamsoft.com wasn't even in the top 25 million recently), so *now* it's adequate to [censor me because of random slander on /r/lisp/](https://twitter.com/HexstreamSoft/status/1308866571557888001), [thereby penalizing up to 128 people](https://twitter.com/HexstreamSoft/status/1309155475791982592).

**Pull request merged**